packages/capacitor-plugin/src/definitions.ts

```diff
@@ -0,0 +1,565 @@
+
+export enum Directory {
+  /**
+   * The Documents directory.
+   * On iOS it's the app's documents directory.
+   * Use this directory to store user-generated content.
+   * On Android it's the Public Documents folder, so it's accessible from other apps.
+   * It's not accesible on Android 10 unless the app enables legacy External Storage
+   * by adding `android:requestLegacyExternalStorage="true"` in the `application` tag
+   * in the `AndroidManifest.xml`.
+   * On Android 11 or newer the app can only access the files/folders the app created.
+   *
+   * @since 1.0.0
+   */
+  Documents = 'DOCUMENTS',
+
+  /**
+   * The Data directory.
+   * On iOS it will use the Documents directory.
+   * On Android it's the directory holding application files.
+   * Files will be deleted when the application is uninstalled.
+   *
+   * @since 1.0.0
+   */
+  Data = 'DATA',
+
+  /**
+   * The Library directory.
+   * On iOS it will use the Library directory.
+   * On Android it's the directory holding application files.
+   * Files will be deleted when the application is uninstalled.
+   *
+   * @since 1.0.0
```

Comment:
Because this plugin is going to continue being `@capacitor/filesystem`, we should maintain the same `@since` of the current plugin

```suggestion
   * @since 1.1.0
```